packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -721,14 +721,39 @@ class FileCollection extends DocumentCollection {
     return formatBytes(parseInt(file.size), decimal)
   }
 
-  async downloadArchive(fileIds, notSecureFilename = 'files') {
-    const filename = slugify(notSecureFilename)
-    const href = await this.getArchiveLinkByIds(fileIds, filename)
+  /**
+   * Attributes used for create archive link by ids
+   *
+   * @typedef {object} ArchivePages
+   * @property {string} id - Id of the file
+   * @property {number} page - The page number
```

Comment:
This is only for PDF, right? Could you add this precision?

Reply:
In fact, I'm adding it!